inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_b_fs_yx_fsv16_imad_1x1.cl

```diff
@@ -103,17 +143,87 @@ KERNEL(convolution_gpu_b_fs_yx_fsv16_imad_1x1)(
     uint filter_idx = GET_WEIGHTS_INDEX(out_f, feature_offset, 0, 0, 0);
 
     uint input_idx[CEIL_DIV(OUT_BLOCK_SPATIAL, SIMD)] = { };
+    #ifdef SHOULD_USE_DATA_ZP
+        uint input_x[CEIL_DIV(OUT_BLOCK_SPATIAL, SIMD)] = { };
+        uint input_y[CEIL_DIV(OUT_BLOCK_SPATIAL, SIMD)] = { };
+    #endif
+
     __attribute__((opencl_unroll_hint))
     for (uint os = 0; os < CEIL_DIV(OUT_BLOCK_SPATIAL, SIMD); ++os) {
-        uint input_x = out_x_shuffle[os] * STRIDE_SIZE_X - PADDING_SIZE_X;
-        uint input_y = out_y_shuffle[os] * STRIDE_SIZE_Y - PADDING_SIZE_Y;
-        input_idx[os] = INPUT0_GET_INDEX(out_b, feature_offset, input_y, input_x);
+        #ifdef SHOULD_USE_DATA_ZP
+            input_x[os] = out_x_shuffle[os] * STRIDE_SIZE_X - INPUT0_PAD_BEFORE_SIZE_X;
```

Comment:
you should use PADDING_SIZE_X to calculate input coords

Reply:
Fixed